app/views/layouts/_loading_page.html.erb

```diff
@@ -0,0 +1,32 @@
+<script>
+	onload="myFunction()" style="margin:0;">
```

Comment:
Ah hmm, perhaps this has a copy-paste error? it seems to be supposed to be in an HTML tag? 

After all these changes, would one of you mind doing a few screencast gifs to demonstrate this working? Thanks a lot, and I appreciate your patience as we're extra careful with changes that will affect every page in the site. Thanks!!!

Reply:
@jywarren thank you for looking over the code 

@oorjitchowdhary would you like to make the updates and add the gifs?